.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+Thanks for contributing to the `fabric-chaincode-evm` project!
+
+Please make sure you have done the following before submitting:
+
+* [ ] Ran the basic checks: `make basic-checks`
+
+* [ ] Ran the unit tests: `make unit-test`
+
+* [ ] Ran the integration tests: `make integration-test`
+
+* [ ] Added the associated JIRA ticket number in the commit message title
+
+* [ ] Link to the JIRA Ticket:
```

Comment:
I don't think the checkboxes are a good idea. 

I also don't want to really look at this big list every time I make a PR.

We should link to a contributing/dev guide, if we have one, which should contain this information.

Reply:
Last two bullets are kind of the same thing.

Reply:
Disagree about the checkboxes. I think it forces submitters to look at each step. I think a dev guide is more likely to be ignored.

In the last bullet, I want them to actually link the JIRA in the pull request. Since we have no jira github integrations for right now, having the number doesn't automatically get us the link. It is not difficult to find after we have the number, but its more associating PR to the item than  specific commits.

Reply:
And I'm more likely to read a short message.

With this, I'm likely to `CTRL-A backspace` and move on.

Reply:
the only thing you need to do is link the jira link. That is one thing I already miss from gerrit.
Otherwise after you make the comment it is easy to check all the boxes.